.github/actions/run-dagger-pipeline/action.yml

```diff
@@ -0,0 +1,55 @@
+name: "Run Dagger pipeline"
```

Comment:
@alafanechere What do you think about the dagger action? If your into it I can update the other dagger workflows

Reply:
I think it's a nice initiative. 
I think @cpdeethree would like to eventually have a single workflow which will run on every PR.
The logic of what pipeline runs would be inferred by the git diff in the entrypoint.
I we go down this way declaring an action could be overkill, but I like it for our current way of doing things.

Reply:
Im thinking Ill add a few more commands before conors repo is ready.

Should I just use this action for my new commands then and leave the existing ones alone to avoid refactoring twice?